### PR TITLE
Remove desktop and add camel case

### PR DESCRIPTION
**Pull request by CCAyl**
this removes the word "desktop" from our token names, and adds camel case.

are the tokens named what we expect?

### CODE REVIEW

**Comment by CCAyl** [comment]
curious if we want to be more specific for what we include in our typography tokens, because we have all our "atom tokens" and all our `h1` tokens that are built from that, and we are exporting both which end up being duplicates. for example we have:

```
export const weightHeading = 700;
export const lineHeightXlarge = "40px";

export const h1FontWeight = 700;
export const h1LineHeight = "40px";
```

are there situations were we actually need to use the "atom" version vs the `h1` versions?

**Comment by alexandra-lim** [comment]
That's a really good call out Cath. Agree with Ian, my first instinct is that both would be better for the exact same scenario he outlined. It would be closer to the current pattern we have as well with `typography.ts`.

**Review by alexandra-lim — APPROVED** [review]

**Comment by CCAyl** [comment]
all sounds good and makes sense to me! +1 for alex's auggestion in a follow up

File: build/ts/typography.ts [hunk 1]
```diff
@@ -1,117 +1,117 @@
...
+export const sizeCaption = "12px";
+export const sizeMobileH1 = "32px";
+export const sizeMobileH2 = "28px";
+export const sizeMobileH3 = "22px";
+export const lineHeightXlarge = "40px";
+export const lineHeightLarge = "32px";
+export const lineHeightNormal = "24px";
+export const lineHeightSmall = "16px";
+export const fontFamilySansSerif = "Mulish";
+export const weightHeading = 700;
```
[the remark this answers is not in the document]
Comment (iryanclarke) [reply build/ts/typography.ts:25]: I see `weightBold` in this file, unless it was added after this comment 😛

[the remark this answers is not in the document]
Comment (alexandra-lim) [reply build/ts/typography.ts:25]: Oh yes you're right! I missed that

**Review by iryanclarke — APPROVED** [review]

File: build/ts/typography.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexandra-lim) [reply build/ts/typography.ts:25]: Same :joy:

**Pull request merged**